spec/schemas/_common.aggregations.yaml

```diff
@@ -779,6 +779,8 @@ components:
               type: number
           required:
             - doc_count
+          additionalProperties:
+            $ref: '#/components/schemas/Aggregate'
```

Comment:
Adds support for the following response if using aggregations within bucket aggregations

```
aggregations:
  jobAdsWithHiddenSalary:
    doc_count: 2
    numOfHirers:
      value: 2
```